### PR TITLE
allow custom_columns to remain sortable if they were

### PR DESCRIPTION
**Pull request by joshrabinowitz**

### CODE REVIEW

**Comment by joshrabinowitz** [comment]
in progress.

File: lib/Dancer/Plugin/SimpleCRUD.pm [hunk 1]
```diff
@@ -1308,11 +1309,17 @@ SEARCHFORM
     }
 
     my @custom_callbacks = ();
+    my @all_cols = map { $_->{COLUMN_NAME} } @$columns;
     for my $custom_col_spec (@{ $args->{custom_columns} || [] } ) {
-        push @custom_callbacks, {
-            column=>$custom_col_spec->{name}, 
-            transform=> ($custom_col_spec->{transform} or sub { return shift;}),
-        };
+        # don't re-show columns that are already displayed
```
Comment (bigpresh) [inline lib/Dancer/Plugin/SimpleCRUD.pm]: I do wonder if this is the desired behaviour, or if a custom_column whose name matches that  of a column that would have been in the results should trump the column from the DB, instead, so you could e.g. override a column but keep the same name?

**Comment by joshrabinowitz** [comment]
Hm. My thinking is that if the column would have already been displayed, then use the transform from the custom column in that column's display, and don't add the custom column separately at the right of the table. 

So I agree about trumping but would prefer that the overridden column stay in place. 

So we're in agreement except (I think) about where the overridden custom column should appear. Right? Thoughts?

**Comment by joshrabinowitz** [comment]
Will reopen when I have a solution I like.

**Comment by joshrabinowitz** [comment]
See #85

**Pull request closed without merging**